temp.cpp

```diff
@@ -0,0 +1,11 @@
+#include <iostream>
+using namespace std;
+int main(){
+int sguare_side;
```

Comment:
Не хватает отступов внутри блока кода функции main
```suggestion
    int sguare_side;
```